styles/styles.scss

```diff
@@ -18,3 +18,16 @@ body {
     line-height: 1.5;
     color: #212529;
 }
+
+figure {
+    text-align: center;
+
+    /* For smaller images that don't need to be full-width: */
+    &.small > img {
+        max-width: 75%;
+    }
+}
+
+figcaption {
+    font-style: italic;
+}
```

Comment:
I added this to style the figures and their captions.

The `small` class is for the smaller screenshots that would appear too large if they were full-width.

Reply:
While I understand this is a small change in CSS, it is really important that all code changes are discussed with @katjuell as they need to be evaluated. This is not an ideal solution for mobile screens

Reply:
This is why I had tagged @katjuell on this PR, to get input and feedback on the proposed solution. I've now removed this CSS and I've removed all captions as you requested.

Reply:
Changes to code need to be scoped and evaluated, even small changes. Thank you